_posts/0000-01-02-deshegs.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![image](https://user-images.githubusercontent.com/85591837/121272031-94a7d080-c88a-11eb-9869-3db288623ea5.png)

When you find out the Lakers have been eliminated